opennms-doc/guide-install/src/asciidoc/text/opennms/rhel.adoc

```diff
@@ -57,6 +57,24 @@ With the successful installed packages the _OpenNMS_ platform is installed in th
    └── system
 ----
 
+[[gi-install-opennms-rhel-disable-yum-repo]]
+==== Disable Automatic Updates (Optional)
+
+We recommend you disable the OpenNMS Yum repository to avoid upgrades while OpenNMS is running.
+
+OpenNMS requires some manual steps upon upgrade to make sure the database and configuration are consistent in the new version, and on systems with many nodes or lots of events, this can take hours.  For this reason, it is recommended to exclude the OpenNMS packages from yum except when you are planning on performing an upgrade.
```

Comment:
We should use `_{opennms-product-name}_` instead of OpenNMS.
New line for each sentence to make diff fixes easier.